Sources/MongoSwiftSync/ChangeStream.swift

```diff
@@ -29,8 +29,8 @@ public class ChangeStream<T: Codable>: CursorProtocol {
      * This change stream will be dead if `next` returns `nil` or an error. It will also be dead if `tryNext` returns
      * an error, but will still be alive if `tryNext` returns `nil`.
      */
-    public var isAlive: Bool {
-        return self.asyncChangeStream.isAlive
+    public func isAlive() throws -> Bool {
```

Comment:
this throws now, but the only case I can think of that it would throw in is if you closed the client and then called this method.
which is actually true for basically all our methods but we don't document that anywhere else, so I wasn't sure if it was worth mentioning.

Reply:
What about just catching the error and returning false?